main.py

```
@@ -1,5 +1,11 @@
+"""애플리케이션 진입점"""
+
+import uvicorn
+
+
 def main():
-    print("Hello from folioo-ai!")
+    """개발 서버 실행"""
+    uvicorn.run("app.main:app", host="0.0.0.0", port=8000, reload=True)
```

Ruff (0.14.14)

[error] Possible binding to all interfaces

(S104)